### PR TITLE
Netperf benches

### PR DESCRIPTION
**Pull request by Stevenjin8**
Work for my intern project. Some tools to test network performance using netperf with various mesh configurations. Not where or how to insert this.

@keithmattix

### CODE REVIEW

**Comment by Stevenjin8** [comment]
Tests are failing from upstream autopep8 https://github.com/hhatto/autopep8/issues/689. There are several PRs fixing it, but don't seem to be moving forward. Maybe we should pin our version of flake8 to an earlier one? or just use black.

**Comment by keithmattix** [comment]
I'm no Pythonista, but maybe it makes sense to use the linter that seems more future proof?

**Comment by kfaseela** [comment]
wondering how my other PR passed the gencheck, however locally I get the error mentioned here. https://github.com/istio/istio/pull/46518 . Any idea which flake8 version works?

**Comment by Stevenjin8** [comment]
I think the temporary fix is to pin the pycodestyle version to 2.10.0. Relevant issue: https://github.com/hhatto/autopep8/issues/689

**Comment by Stevenjin8** [comment]
> wondering how my other PR passed the gencheck, however locally I get the error mentioned here. [istio/istio#46518](https://github.com/istio/istio/pull/46518) . Any idea which flake8 version works?

@kfaseela Did your other PRs involve python files? I think it will only lint files that have changed.

**Comment by kfaseela** [comment]
> > wondering how my other PR passed the gencheck, however locally I get the error mentioned here. [istio/istio#46518](https://github.com/istio/istio/pull/46518) . Any idea which flake8 version works?
> 
> @kfaseela Did your other PRs involve python files? I think it will only lint files that have changed

Then I am.not sure why local run fails when I have not changed any python files

**Comment by keithmattix** [comment]
@Stevenjin8 The errors I'm seeing from prow now should be fixed with a `make gen`

**Comment by ericvn** [comment]
LGTM. @AndreaM12345 Any thoughts?

**Review by keithmattix — COMMENTED** [review]
LGTM as well

File: ambient-tests/README.md [hunk 1]
```diff
@@ -28,12 +31,12 @@ I don't automate this process because there are many installation methods and yo
 To install the latest release of Istio Ambient [install `istioctl`](https://istio.io/latest/docs/setup/getting-started/#download) and run
 
 ```bash
-istiocl install --set profile=ambient
+istioclt install --set profile=ambient
```
Comment (ericvn) [inline ambient-tests/README.md:34]:
```suggestion
istioctl install --set profile=ambient
```

**Review by ericvn — APPROVED** [review]

**Pull request merged**